### PR TITLE
New article 'r2dii.plot A: Thin API'

### PR DESCRIPTION
**Pull request by maurolepore**
This PR aims to add articles for the "X" and "Y" versions of the API.

* [Index](https://2degreesinvesting.github.io/r2dii.plot/reference/index.html)
* [Indroduction to version X](https://2degreesinvesting.github.io/r2dii.plot/articles/articles/r2dii-plot-X.html)
* [Indroduction to version Y](https://2degreesinvesting.github.io/r2dii.plot/articles/articles/r2dii-plot-Y.html)

### CODE REVIEW

**Comment by maurolepore** [comment]
@MonikaFu, I still plan to write two articles: (1) about the think API and (2) comparing the thin vs. think API.

I took the liberty to publish the WIP to the website so you can see the end result conveniently. See the URLs at the top.

File: vignettes/articles/r2dii-plot-A.Rmd [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+plot_timelineX(sda)
+```
+
+The error message suggests you must first pick only one value of `sector`, for
+example "cement".
+
+```{r}
+sda %>% 
+  filter(sector == "cement") %>% 
+  plot_timelineX()
```
Comment (MonikaFu) [inline vignettes/articles/r2dii-plot-A.Rmd:69]: I see in the output that the function doesn't filter the starting year (all lines starting in the same year). Should we add it? The current output doesn't look correct.

File: vignettes/articles/r2dii-plot-A.Rmd [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+unique(market_share$metric)
+chosen_metrics <- c("projected", "corporate_economy", "target_sds")
+
+market_share %>%
+  filter(
+    metric %in% chosen_metrics,
+    sector == "power",
+    region == "global"
+  ) %>% 
+  plot_techmixX() + labs(title = "power")
```
Comment (MonikaFu) [inline vignettes/articles/r2dii-plot-A.Rmd:104]: Can we capitalise the title here? It hurts me to see it like that 😆

File: vignettes/articles/r2dii-plot-A.Rmd [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+each sector with `lapply()`.
+
+```{r}
+plot_sector_techmix <- function(x) {
+  chosen_metrics <- c("projected", "corporate_economy", "target_sds")
+  
+  market_share %>%
+    filter(
+      metric %in% chosen_metrics,
+      sector == x,
```
Comment (MonikaFu) [inline vignettes/articles/r2dii-plot-A.Rmd:118]: Here we could use `sector == to_lower(x)` or sth similar with a capitalised title (see comment above).

File: vignettes/articles/r2dii-plot-A.Rmd [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+```{r}
+market_share %>% 
+  filter(
+    sector == "power", 
+    region == "global", 
+    technology == "renewablescap"
+  ) %>% 
+  plot_trajectoryX()
+```
```
Comment (MonikaFu) [inline vignettes/articles/r2dii-plot-A.Rmd:143]: I'm curious why the annotations do not get the lines as they did in my version of the plot. Something to investigate.

**Comment by MonikaFu** [comment]
> @MonikaFu, I still plan to write two articles: (1) about the think API and (2) comparing the thin vs. think API.
> 
> I took the liberty to publish the WIP to the website so you can see the end result conveniently. See the URLs at the top.

Did you mean thin vs. thick?

**Comment by MonikaFu** [comment]
Overall the thin API article is looking good! I like the explanations of the workflow. Seem clear and easy to follow.

**Comment by maurolepore** [comment]
@MonikaFu, 
Thanks a lot for your comments! I'll work on that Monday. In the meantime, you may want to see the first bit of [r2dii.plot B: Thick API.](https://2degreesinvesting.github.io/r2dii.plot/articles/articles/r2dii-plot-B-Thik-API.html).

The A/B from the thin/thick API may be confused with the A/B/C versions of some functions within the thick API. To minimize the confusion I'm thinking of renaming the articles to X/Y instead of A/B (or something else).

**Comment by MonikaFu** [comment]
@maurolepore - for thick API I have again the same comment as for the thin - that the timelines are not starting at a common year and that it does not look good I think. Did we lose this feature somewhere on the way?

File: vignettes/articles/r2dii-plot-A.Rmd (hunk 2 above)
Comment (maurolepore) [reply vignettes/articles/r2dii-plot-A.Rmd:104]: I'll remove the iteration and remove the title of the single plot we show

**Pull request closed without merging**